install_common_crypto.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+FRAMEWORK_DIR="${BUILT_PRODUCTS_DIR}/CommonCrypto.framework"
+
+# Check if module map exists (Xcode 10). If not, generate a dummy module.
+
+# Skip if directory already exists.
+if [ -d "${FRAMEWORK_DIR}" ]; then
+    echo "${FRAMEWORK_DIR} already exists, so skipping the rest of the script."
+    exit 0
+fi
+
+mkdir -p "${FRAMEWORK_DIR}/Modules"
+cat <<EOF > "${FRAMEWORK_DIR}/Modules/module.modulemap"
+module CommonCrypto [system] {
+    header "${SDKROOT}/usr/include/CommonCrypto/CommonCrypto.h"
+    export *
+}
+EOF
+
+ln -sf "${SDKROOT}/usr/include/CommonCrypto" "${FRAMEWORK_DIR}/Headers"
```

Comment:
Isn't this a bit "intrusive"? 🤔 
Also, if other libs do the same "hack", this will break quite easily. Or am I missing something?

Reply:
We're just linking the framework module we just created (`FRAMEWORK_DIR` points to our `CommonCrypto.framework`). It shouldn't break since we only proceed if the framework doesn't exist.

Reply:
I understand that we're pointing to our dummy module, and we only do it if it doesn't exist. My concerns are:

1. we're creating a link inside the user's `$SDKROOT` which I think is a bit intrusive.
2. any other lib making the same "hack" will overwrite whatever is at `${SDKROOT}/usr/include/CommonCrypto` to point to their own dummy module, making "our" link broken in the process, while the check for the existence of the module keeps passing.

Reply:
Hm, doesn't `ln` work the other way around – `ln -s /path/to/original/ /path/to/link`?

Reply:
Oops, you're right 😅 
Please ignore my "concerns"